code/modules/mining/equipment/mining_tools.dm

```diff
@@ -85,24 +94,36 @@
 
 /obj/item/pickaxe/drill/diamonddrill
 	name = "diamond-tipped mining drill"
+	desc = "Yours is the drill that will pierce the heavens!"
 	icon_state = "diamonddrill"
 	toolspeed = 0.2
-	desc = "Yours is the drill that will pierce the heavens!"
+	force = 20
+	armour_penetration = -10
 
 /obj/item/pickaxe/drill/cyborg/diamond //This is the BORG version!
 	name = "diamond-tipped cyborg mining drill" //To inherit the NODROP_1 flag, and easier to change borg specific drill mechanics.
+	desc = "Yours is the drill that will pierce the heavens!" // Copied from the regular diamond drill.
 	icon_state = "diamonddrill"
 	toolspeed = 0.2
+	force = 20
+	armour_penetration = -10
 
 /obj/item/pickaxe/drill/jackhammer
 	name = "sonic jackhammer"
+	desc = "Cracks rocks with sonic blasts."
 	icon_state = "jackhammer"
 	inhand_icon_state = "jackhammer"
 	worn_icon_state = "jackhammer"
 	toolspeed = 0.1 //the epitome of powertools. extremely fast mining
 	usesound = 'sound/weapons/sonic_jackhammer.ogg'
 	hitsound = 'sound/weapons/sonic_jackhammer.ogg'
-	desc = "Cracks rocks with sonic blasts."
+	attack_verb_simple = list("hit", "pierce", "mine", "slam", "attack")
+	attack_verb_continuous = list("hits", "pierces", "mines", "slams", "attacks")
+	force = 10
+	armour_penetration = 50
+	sharpness = SHARP_NONE
+	wound_bonus = 40
+	bare_wound_bonus = 60 // It is pumping rock-shattering levels of ultrasound directly into your body. That's going to do _nasty_ things to your bones.
 
 /obj/item/pickaxe/improvised
 	name = "improvised pickaxe"
```

Comment:
I was debating whether or not to mention this, but improvised pickaxes will inherit the wound-related stuff from normal pickaxes.

is that okay?

Reply:
I'm reasonable okay with that. even if they are improvised they are still pickaxes after all.

Reply:
Alright, I made the improvised pickaxe a bit closer to a kitchen knife.